### PR TITLE
Rework donate button

### PR DESCRIPTION
**Pull request by Vincevrp**
### Description

Reworked donated button to improve looks and fix W3 error. Small changes to nav to improve responsiveness as well. 

### HTML Preview

https://htmlpreview.github.io/?https://github.com/Vincevrp/privacytools.io/blob/donate-button/index.html

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -17,6 +17,25 @@ body {
...
+}
+
+@media (max-width: 992px) {
+  nav > .container {
+    width: 100%;
+  }
+}
+
+#donate {
+  float: right;
```
Comment (Shifterovich) [inline css/custom.css]: The `float: right` gets ugly when navbar is collapsed.

screenshot from 2018-10-23 15-25-34

File: css/custom.css [hunk 2]
```diff
@@ -17,6 +17,25 @@ body {
...
+}
+
+#donate > a {
+  padding: 15px 0;
```
Comment (Shifterovich) [inline css/custom.css]: I'd add some space (margin) between the text and the heart.

**Comment by Vincevrp** [comment]
> Maybe we should open a big PR / branch to change the look to BS4? Now that websites use BS4, BS3 is quite ugly.

Maybe we should implement some sort of CI/CD with linting and validation before migrating to BS4? Just to make sure that the future code will be W3 compliant.

**Comment by Vincevrp** [comment]
> > Maybe we should open a big PR / branch to change the look to BS4? Now that websites use BS4, BS3 is quite ugly.
> 
> Maybe we should implement some sort of CI/CD with linting and validation before migrating to BS4? Just to make sure that the future code will be W3 compliant.

I will fix this issues later this week.

**Comment by Shifterovich** [comment]
@Vincevrp Good idea.

We really should switch to BS4. I could help with some of these things better as I'm very familiar with BS4 syntax.

**Comment by Shifterovich** [comment]
I'll start some work on the transition already. I'll create a new branch.

**Comment by Vincevrp** [comment]
Should be fixed now.

**Comment by Shifterovich** [comment]
If we're going to merge the BS4 branch soon, this change is unnecessary. See the BS4 version of the navbar:

screenshot from 2018-10-25 17-11-33

**Comment by Vincevrp** [comment]
I finished it so it's fixed while BS4 is getting implemented. If you prefer not to merge it you may close this PR.

**Comment by Shifterovich** [comment]
I think that's @privacytoolsIO's BTC address, but I'm not even sure if he still has access to it, I haven't heard from him for a long time.

> Your donation will cover our cost for server, domains, coffee, beer and pizza.

Right now our (@privacytoolsIO's) only expense is the domain cost (and possibly Cloudflare, not sure). Which is covered for many years ahead from the BTC on these two addresses only https://www.blockchain.com/btc/address/1N4bKCx3kpvUTBhsvbmthPDssN8ATPEhgt https://www.blockchain.com/btc/address/1EM2JEbQeoiZP4Cd4Pm2wgJcBPAWJBeijz (there's also https://www.blockchain.com/btc/address/1N6heMWD34ARyApkRmNv7V7NzQfYvgC4dg which had a few BTC prior to the creation of PTIO). I'd personally rather see any money left after domain costs are paid invested in other privacy projects.

**Pull request merged**